better_profanity/better_profanity.py

```diff
@@ -62,8 +61,12 @@ def add_censor_words(self, custom_words):
             raise TypeError(
                 "Function 'add_censor_words' only accepts list, tuple or set."
             )
-
-        self.CENSOR_WORDSET.update(custom_words)
+        # Convert all arguments to lower case. Works for tuples and lists too from re-casting into sets
+        custom_words = {word.lower() for word in custom_words}
```

Comment:
You loop all the words here, and then loop them again two lines down. I would just convert the word to lowercase in the loop below

Reply:
good catch. ooof i wonder why i didnt realize that at the moment